.github/workflows/JirabotMerge.yml

```diff
@@ -254,14 +254,17 @@ jobs:
                 print('Error: PROJECT_CONFIG is missing required fields: tagPrefix and/or tagPostfix')
                 sys.exit(1)
 
-            project_name = projectConfig.get('projectName')
-            if project_name is None:
-                print('Error: PROJECT_CONFIG is missing required field: projectName')
+            project_prefixes = projectConfig.get('projectPrefixes')
+            if project_prefixes is None:
+                print('Error: PROJECT_CONFIG is missing required field: projectPrefixes')
                 sys.exit(1)
 
+            project_list_regex = '|'.join(project_prefixes)
```

Comment:
I'm assuming PROJECT_CONFIG can now be comprised of multiple project names delimited by '|' ?  It might be worth explicitly defining the newly supported syntax somewhere. 
Also, perhaps outside the scope of this change but the Errors regarding  PROJECT_CONFIG could be more informative about what PROJECT_CONFIG is (github action variable)